src/humam/data_preprocessing/connectivity.py

```diff
@@ -1,13 +1,8 @@
-# -*- coding: utf-8 -*-
 import numpy as np
-from scipy import stats, integrate
+import scipy as sci
 import pandas as pd
-from itertools import product
-from data_loader.microcircuit import p as p_PD
-from data_loader.microcircuit import K_ext as K_ext_PD
-from data_loader.hcp_dti import HcpDesikanKilliany, VolumesDK
-from data_loader.synapse_cellbody_probability import mohan, binzegger
 
+from .. import data_loader as dl
```

Comment:
```suggestion
import humam.data_loader as dl
```
relative import upwards can become problematic.